digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala

```diff
@@ -1199,6 +1214,47 @@ class Digiroad2Api(val roadLinkService: RoadLinkService,
       }.getOrElse(Map("success:" ->false, "Reason"->"Id not found or invalid input"))
   }
 
+  get("/municipalities/unverified") {
+    val user = userProvider.getCurrentUser()
+    val municipalities: Set[Int] = if (user.isOperator()) Set() else user.configuration.authorizedMunicipalities
+    assetService.getMunicipalitiesNameByCode(municipalities)
+
+  }
+
+  get("/municipalities/assetTypes/:municipalityCode") {

```

Comment:
Is there any reason for municipalities end points can not be like this: 

`/municipalities/:municipalityCode/...`